go.mod

```diff
@@ -9,8 +9,12 @@ exclude k8s.io/kubernetes v1.13.0
 replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.5
 
 require (
```

Comment:
This file (go.mod) should not be updated in this PR.

Maybe it's because you ran `make gen`? I would suggest running `make snips` instead, since that will update the snips but nothing else that may be out of sync in your branch.